src/community/__stories__/blocks-story.tsx

```diff
@@ -133,8 +133,10 @@ type HighlightedValueBlockArgs = {
 
     secondaryValue: string;
 
+    Pretitle: string;
```

Comment:
```suggestion
    pretitle: string;
```

uppercase inconsistency